lib/srv/db/server.go

```diff
@@ -585,7 +586,10 @@ func (s *Server) Close() error {
 // Wait will block while the server is running.
 func (s *Server) Wait() error {
 	<-s.closeContext.Done()
-	return s.closeContext.Err()
+	if err := s.closeContext.Err(); err != nil && err != context.Canceled {
+		return err
```

Comment:
trace.Wrap?

Reply:
Sure, was trying to maintain current behavior as much as possible, but that's an easy low-risk change.